src/styles/blocks/card.scss

```diff
@@ -0,0 +1,130 @@
+.card {
+  height: 406px;
+  width: 200px;
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/cca19e86-0f73-4a3e-957f-032c4c7cdbd8)

Reply:
Not fixed